packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx

```diff
@@ -38,46 +42,91 @@ export const NotificationsWidget: FC<NotificationsWidgetProps> = ({ ...rest }) =
   }
 
   return (
-    <Popover hideOnClick ref={popoverRef} {...rest} onShow={handleShow} onHide={handleHide}>
-      <Wrapper>
-        <Header>
-          <Text as="h3" variant="h400">
-            Notifications
-          </Text>
-          <Button variant="secondary" size="small" onClick={() => markNotificationsAsRead(notifications)}>
-            Mark all as read
-          </Button>
-        </Header>
-        <Content>
-          {notifications.length > 0 ? (
-            notifications.map((notification, idx) => (
-              <NotificationTile
-                variant="compact"
-                key={`notification-${notification.id}-${idx}`}
-                notification={notification}
-                onClick={() => {
-                  popoverRef.current?.hide()
-                  markNotificationsAsRead(notification)
+    <>
+      {!smMatch && isOpen && <MobileBackdrop onClick={() => handleHide()} />}
```

Comment:
```suggestion
      {!smMatch && isOpen && <MobileBackdrop  />}
```
Worth to double check this, but I'm pretty sure that we don't need this onClick handler. Popover should handle this